pkgs/tools/networking/miniupnpc/default.nix

```diff
@@ -27,8 +27,8 @@ let
     };
 in {
   miniupnpc_2 = generic {
-    version = "2.0.20171212";
-    sha256 = "0za7pr6hrr3ajkifirhhxfn3hlhl06f622g8hnj5h8y18sp3bwff";
+    version = "2.0.20180203";
```

Comment:
any reason this version in particular ?

Reply:
Right, I should have written about this.  I'm being somewhat conservative here.  This isn't the latest version of miniupnpc; however it is the version of miniupnpc that Bitcoin Core currently ships with (see https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md).  I'm not opposed to moving to the latest, but this was the minimum version bump that is needed to bring it inline with what Bitcoin Core is using.

Reply:
👍